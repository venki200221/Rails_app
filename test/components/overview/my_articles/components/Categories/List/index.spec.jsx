import React from 'react';
import { shallow } from 'enzyme';
import '../../../../../../testHelper';

import List from '@components/overview/my_articles/components/Categories/List/List.jsx';

describe('List', () => {
  it('renders', () => {
    const props = {
      assignments: [{ id: 1 }, { id: 2 }],
      course: {},
      current_user: {},
      title: 'Article Title',
      wikidataLabels: {}
    };

    const component = shallow(<List {...props} />);
    expect(component).toMatchSnapshot();
  });
});
