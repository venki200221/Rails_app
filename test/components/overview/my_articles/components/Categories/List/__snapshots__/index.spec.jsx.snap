// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders 1`] = `ShallowWrapper {}`;
